examples/mugen/test/retrieval/test_video_clip.py

```diff
@@ -4,11 +4,48 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import os
+from typing import Optional
+
 import pytest
 import torch
 from examples.mugen.retrieval.video_clip import TextEncoder, VideoEncoder
 
-from test.test_utils import assert_expected, set_rng_seed
+from test.test_utils import assert_expected, get_asset_path, set_rng_seed
+from torchmultimodal.utils.common import get_current_device, shift_dim
+
+
+def patch_load_model(mocker):
+    """Mock the ``load_model`` function of ``PretrainedMixin`` to allow loading truncated state dicts with ``strict=False``."""
+
+    def patched_load_model(
+        cls,
+        pretrained_url: Optional[str],
+        load_state_dict: bool = True,
+        state_dict_key: Optional[str] = None,
+    ):
+        assert isinstance(
+            cls, torch.nn.Module
+        ), "load_model can only be called on an nn.Module instance"
+        if os.path.exists(pretrained_url):
+            state_dict = torch.load(pretrained_url, map_location=get_current_device())
+        else:
+            state_dict = torch.hub.load_state_dict_from_url(
+                pretrained_url,
+                model_dir=cls.get_model_dir(pretrained_url),
+                map_location=get_current_device(),
+            )
+        if state_dict_key:
+            state_dict = state_dict[state_dict_key]
+
+        if load_state_dict:
+            cls.load_state_dict(state_dict, strict=False)
+        return state_dict
+
+    return mocker.patch(
+        "torchmultimodal.utils.common.PretrainedMixin.load_model",
```

Comment:
I can't use `load_state_dict` to patch `load_state_dict` because it throws a RecursionError (similarly, I couldn't think of a way to shorten the copy paste code when i mocked `load_model`, since i couldn't use the existing `load_model`). 

I'll change the path to the VideoEncoder path, but not specifying the whole path also throws a ModuleNotFoundError